internal/database/database.go

```diff
@@ -0,0 +1,211 @@
+package database
+
+import (
+	"database/sql"
+	"final-project/internal/models"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+func InitDB() (*sql.DB, error) {
+	dbFile := os.Getenv("TODO_DBFILE")
+	if dbFile == "" {
+		appPath, err := os.Executable()
+		if err != nil {
+			return nil, err
+		}
+		dbFile = filepath.Join(filepath.Dir(appPath), "scheduler.db")
+	}
+	_, err := os.Stat(dbFile)
+	install := os.IsNotExist(err)
+
+	db, err := sql.Open("sqlite", dbFile)
+	if err != nil {
+		return nil, err
```

Comment:
Лучше ошибки оборачивать, чтобы быстрее понимать, где произошла ошибка
```suggestion
		return nil, fmr.Errorf("error while open db: %w", err)
```